src/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -33,10 +33,8 @@
 import games.strategy.common.swing.SwingComponents;
 import games.strategy.util.Version;
 
-
```

Comment:
There is going to be a merge conflict with this change depending upon who merges first.. FYI, no action to be taken.